zeroex/asset_data_decoder.go

```diff
@@ -55,10 +68,21 @@ type ERC1155AssetData struct {
 type ERC20BridgeAssetData struct {
 	TokenAddress  common.Address
 	BridgeAddress common.Address
-	Calldata      []byte
+	BridgeData    []byte
+}
+
+// StaticCallAssetData represents a StaticCallAssetData
```

Comment:
nit: I know this PR isn't the first time this type of comment showed up, but this doesn't add any useful information and IMO we'd be better off removing it. Alternatively we can replace it with a comment that provides some more context.